src/ConformalTracking.cc

```diff
@@ -913,7 +920,10 @@ void ConformalTracking::combineCollections(SharedKDClusters& kdClusters, UKDTree
 
   // Loop over all given collections
   for (unsigned int i = 0; i < combination.size(); i++) {
+
     // Copy the clusters to the output vector
+    if( collectionClusters.size() <= unsigned( combination[i] )  ) continue; // protect against empty subdetectors
```

Comment:
This check is wrong, the size of the map doesn't say anything about which key is present.